### ISSUE TITLE
Please verify README.rst

### ISSUE THREAD

**Issue by depaoli** [voronind/range-regex#5]
Hello

and thanks for this module, quite useful!

I'm just noting that <code>from range_regex import regex_for_range</code> statement in the example from README.rst doesn't work, shouldn't it be <code>from range_regex import bounded_regex_for_range</code>?

Thanks
Matteo

**Comment by MalikRumi** [comment]
@depaoli Yes, I can confirm from my own experience just now that you are absolutely correct. I'm using Python 3.5.2, and all I can say is THANK GOD for you and your post, I might not have figured it out without you and thus been unable to use this very valuable (for me, at least) tool.
